sdk/keyvault/azcertificates/client.go

```diff
@@ -0,0 +1,1568 @@
+	if err != nil {
+		return GetIssuerResponse{}, err
+	}
+
+	return GetIssuerResponse{
+		RawResponse: resp.RawResponse,
+		IssuerBundle: IssuerBundle{
+			ID:                  resp.ID,
+			Provider:            resp.Provider,
+			Attributes:          issuerAttributesFromGenerated(resp.Attributes),
+			Credentials:         issuerCredentialsFromGenerated(resp.Credentials),
+			OrganizationDetails: organizationDetailsFromGenerated(resp.OrganizationDetails),
+		},
+	}, nil
+}
+
+// ListIssuersPager is the pager returned by Client.ListIssuers
+type ListIssuersPager struct {
+	genPager *generated.KeyVaultClientGetCertificateIssuersPager
+}
+
+// PageResponse returns the results from the page most recently fetched from the service
+func (l *ListIssuersPager) PageResponse() ListIssuersPage {
+	return listIssuersPageFromGenerated(l.genPager.PageResponse())
+}
+
+// Err returns an error value if the most recent call to NextPage was not successful, else nil
+func (l *ListIssuersPager) Err() error {
+	return l.genPager.Err()
+}
+
+// NextPage fetches the next available page of results from the service. If the fetched page
+// contains results, the return value is true, else false. Results fetched from the service
+// can be evaluated by calling PageResponse on this Pager.
+func (l *ListIssuersPager) NextPage(ctx context.Context) bool {
+	return l.genPager.NextPage(ctx)
+}
+
+// ListIssuersOptions contains the optional parameters for the Client.ListIssuers method
+type ListIssuersOptions struct {
+	MaxResults *int32
+}
+
+// convert ListIssuersOptions to generated options
+func (l *ListIssuersOptions) toGenerated() *generated.KeyVaultClientGetCertificateIssuersOptions {
+	if l == nil {
+		return &generated.KeyVaultClientGetCertificateIssuersOptions{}
+	}
+
+	return &generated.KeyVaultClientGetCertificateIssuersOptions{Maxresults: l.MaxResults}
+}
+
+// ListIssuersPage contains the current page of results for the Client.ListSecrets operation
+type ListIssuersPage struct {
+	// READ-ONLY; A response message containing a list of certificates in the key vault along with a link to the next page of certificates.
+	Issuers []*CertificateIssuerItem `json:"value,omitempty" azure:"ro"`
+
+	// RawResponse contains the underlying HTTP response.
+	RawResponse *http.Response
+}
+
+// convert internal Response to ListIssuersPage
+func listIssuersPageFromGenerated(i generated.KeyVaultClientGetCertificateIssuersResponse) ListIssuersPage {
+	var vals []*CertificateIssuerItem
+
+	for _, v := range i.Value {
+		vals = append(vals, certificateIssuerItemFromGenerated(v))
+	}
+
+	return ListIssuersPage{
+		RawResponse: i.RawResponse,
+		Issuers:     vals,
+	}
+}
+
+// ListIssuers returns a pager that can be used to get the set of certificate issuer resources in the specified key vault. This operation
+// requires the certificates/manageissuers/getissuers permission.
+func (c *Client) ListIssuers(options *ListIssuersOptions) ListIssuersPager {
```

Comment:
We all call this `ListPropertiesOfIssuers` since it returns their properties and not an actual `CertificateIssue`: https://github.com/Azure/azure-sdk-for-net/blob/a7743d7564c9d60dff0ec17822af0f18323f5f15/sdk/keyvault/Azure.Security.KeyVault.Certificates/src/CertificateClient.cs#L1375

Reply:
Fixed